src/debugger.c

```diff
@@ -632,7 +632,7 @@ static int DEBUGRenderZeroPageRegisters(int y) {
 			DEBUGString(dbgRenderer, DBG_ZP_REG, y, lbl, col_label);
 
 			int reg_addr = 2 + reg * 2;
-			int n = real_read6502(direct_page_add(reg_addr+1), true, currentBank)*256+real_read6502(direct_page_add(reg_addr), true, currentBank);
+			int n = debug_read6502(direct_page_add(reg_addr+1), -1)*256+debug_read6502(direct_page_add(reg_addr), -1);
```

Comment:
`-1` should be a constant define like `#define USE_CURRENT_BANK (-1)`, it's not super clear what it means otherwise.